staging/src/k8s.io/apiserver/pkg/admission/decorator.go

```diff
@@ -0,0 +1,22 @@
+package admission
+
+type Decorator interface {
```

Comment:
switched to interface to allow func and struct, which allows the slice or decorators to work nicely in the type system